.changelog/11902.txt

```diff
@@ -0,0 +1,4 @@
+```release-note:bug
+template: Fixed a bug where client template configuration that did not include any
+of the new options could result in the entire struct getting set to nil.
```

Comment:
Generally speaking the changelog should be user-facing. A "struct getting set to nil" doesn't mean anything to the end user, so the changelog should say what user-visible bug this fixes.

Reply:
Good points. I'll get better at this I promise 😸  . I'll update this before I merge.